common/djangoapps/pipeline_mako/templates/static_content.html

```diff
@@ -88,14 +88,19 @@
     </script>
 </%def>
 
-<%def name="optional_include_mako(file, with_microsite=False)"><%
+<%def name="optional_include_mako(file, with_microsite=False, fallback=None, **kwargs)"><%
 # http://stackoverflow.com/q/21219531
 if with_microsite:
     file = microsite.get_template_path(file)
+tmpl = None
 try:
     tmpl = self.get_template(file)
 except TemplateLookupException:
-    pass
-else:
-    tmpl.render_context(context)
-%></%def>
+    if fallback:
+        try:
+            tmpl = self.get_template(fallback)
+        except TemplateLookupException:
+            pass
+if tmpl:
+    tmpl.render_context(context._locals(kwargs))
```

Comment:
@singingwolfboy after the little research and some several attempts I couldn't find the way how to do this. <%include> is a build in function but optional_include_mako isn't. So in case if I use
```
<%static:optional_include_mako file="courseware/tab_${tab.tab_id}.html" fallback="courseware/tab_default.html" with_microsite="True" args="tab=tab,tab_is_active=tab_is_active,tab_class=tab_class" />
```
nothing will be passed into optional_include_mako function.
I've tried to define function optional_include_mako like this:
```
<%def name="optional_include_mako(file, with_microsite=False, fallback=None, args=None, **kwargs)">
```
or like this:
```
<%def name="optional_include_mako(file, with_microsite=False, fallback=None, *args, **kwargs)">
```
but nothing has changed.